src/transformStateWithClones.js

```diff
@@ -7,7 +7,54 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const propertiesOfActios = [];
```

Comment:
The variable name `propertiesOfActios` seems to be a typo and does not clearly describe its purpose. Consider renaming it to `stateHistory` or `statesAfterActions` to better reflect its usage.